src/server/endpoints/loginHandler.js

```diff
@@ -2,31 +2,49 @@
 
 const db = require('../collections/usersDatabase');
 const HTTP_STATUSES = require('../modules/httpStatuses');
+const tokensCollection = require('../collections/tokens.js');
+const deleteAccessToken = tokensCollection.deleteDocumentByToken;
```

Comment:
if you need only one function from the export you can use the destructor notation
`const {deleteAccesToken} = require('../collections/tokens.js');`